pkg/logger/logger.go

```diff
@@ -322,16 +341,22 @@ func (l *Logger) Sync() error {
 var (
 	// Package-level Logger
 	pkgLogger = &Logger{}
+	pkgMutex  = &sync.RWMutex{}
 )
 
 // Current returns the package-level base logger
 func Current() *Logger {
+	pkgMutex.RLock()
+	defer pkgMutex.RUnlock()
```

Comment:
We need to, since any chunk of memory accessed without synchronicity is subject to data race. I had the same doubt, because in this case it is just a pointer copy, but it is not guaranteed to be an atomic read.